content/en/docs/Writing policies/generate.md

```diff
@@ -246,7 +246,111 @@ spec:
           foo: bar
 ```
 
-## Generating resources into existing namespaces
+## Generate for Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports the generate existing resources. Generate existing policies are applied in the background which creates target resource in existing matched resources in the cluster.
+They may also optionally be configured to apply upon updates to the policy itself.
+
+{{% alert title="Note" color="warning" %}}
+The proper permissions may need to be granted to Kyverno ServiceAccount. Kubernetes resources may require granting additional privileges to the ClusterRole responsible for the generate behavior. To enable Kyverno to generate these other types, edit the ClusterRole typically named kyverno:generate and add or update the rules to cover the resources and verbs needed.
+{{% /alert %}}
```

Comment:
This applies to all generate policies, not just "generate for existing". Do we want to add the note in this section? We also clarify this in the installation guide.